Sources/AngouriMath/Core/Parser.cs

```diff
@@ -66,7 +66,8 @@ public static Entity Parse(string source)
                     // 2x -> 2 * x       2sqrt -> 2 * sqrt       2( -> 2 * (
                     // x y -> x * y      x sqrt -> x * sqrt      x( -> x * (
                     // )x -> ) * x       )sqrt -> ) * sqrt       )( -> ) * (
-                    (NUMBER or VARIABLE or PARENTHESIS_CLOSE, VARIABLE or FUNCTION_OPEN or PARENTHESIS_OPEN) => lexer.Multiply,
+                    (NUMBER or VARIABLE or PARENTHESIS_CLOSE, VARIABLE or FUNCTION_OPEN or PARENTHESIS_OPEN) => MathS.Settings.ExplicitParsingOnly ? throw new InvalidArgumentParseException("Cannot Multiply '*' When  MathS.Settings.ExplicitParsingOnly.Set(true)  has been called" + $"\n" +
```

Comment:
Might make sense to split it into lines. My preference is
```cs
(...) =>
condition
? something
: somethingElse
```

Reply:
Alright